PlayCover/Utils/Cacher.swift

```diff
@@ -8,16 +8,42 @@
 import Foundation
 import AppKit
 import DataCache
+import CachedAsyncImage
 
 class Cacher {
 
+    @ImageCache private var imageCache
     let cache = DataCache.instance
     /// We can create a custom cache like this (default values are as the same as below):
     /// `let cache = DataCache(name: "PlayCoverCache")`
     /// `cache.maxDiskCacheSize = 100*1024*1024`      // 100 MB
     /// `cache.maxCachePeriodInSecond = 7*86400`      // 1 week
     /// More details: https://github.com/huynguyencong/DataCache/blob/master/README.md
 
+    init() {
+        // Set image cache limit.
+        ImageCache().wrappedValue.setCacheLimit(
+            countLimit: 400,
+            totalCostLimit: 4*1024*1024
+        )
+    }
+
+    func removeImageCache() {
+        imageCache.removeCache()
```

Comment:
since you are going this path (i dunno if there are other ways) you should remove the URLCache extension in the end of this file since it was for the prev package.